.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all",
+  "semi": true,
+  "bracketSpacing": true,
+  "printWidth": 70,
+  "parser": "flow",
```

Comment:
Prettier default [babylon](https://github.com/babel/babylon) parser is supporting flow, I think not need to change it?